django/contrib/admin/static/admin/img/inline-delete.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="16" height="16" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg">
+<svg width="100%" height="100%" viewBox="0 0 1792 1792" preserveAspectRatio="xMinYMin" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg">
```

`width` and `height` default to 100% so I don’t think we need them. And `preserveAspectRatio` isn’t necessary since the viewBox aspect ratio is square and we only ever want to display this icon with a square aspect ratio.